src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
```

Comment:
Consider using 'this' instead of the 'calculator' reference when referring to the current instance within the object methods. This will make your methods more robust, especially if the object is assigned to a different variable or the method is used as a callback. Example: this.result += number;